### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JYOSHREDDY**
features of Geoweaver added in the README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ Geoweaver is an in-browser software allowing users to easily compose and execute
 
 It can be run from both local and remote (distributed) machines.
 
+Why choose Geoweaver?
+1) Safely Store all your progress along the way.
+2) Visualise your results using the tools that you already use.
+3) Stay organised through out your research
+4) Supports CRON executions
```
Comment (ZihengSun) [inline README.md:13]: @JYOSHREDDY can you expand what CRON is?

**Comment by gokulprathin8** [comment]
I can jointly work on this with @JYOSHREDDY 
to redo the entire README to make it a bit more modern.

Please reach out to me at your convenient time @ slack to discuss and improve the overall readme file.

**Comment by gokulprathin8** [comment]
Here is the outline we discussed about
https://docs.google.com/document/d/1ATng4rOhrOTIGIEGprWgKy7MAbm0yz_AGkmtCDhPEZ8/edit?usp=sharing

please feel free to make changes if you feel there is a better way of doing it
also remove Circle CI passing badge and needs to be changed to Github Actions
since we are no longer using circle ci

**Comment by ZihengSun** [comment]
@gokulprathin8 yeah, that is good. Although could you please help @JYOSHREDDY to merge this PR first and file another issue to completely refactor the README. It think that one will take longer than this PR

**Comment by gokulprathin8** [comment]
ok

File: README.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ Geoweaver is an in-browser software allowing users to easily compose and execute
 
 It can be run from both local and remote (distributed) machines.
 
+Why choose Geoweaver?
+1) Safely Store all your progress along the way.
+2) Visualise your results using the tools that you already use.
+3) Stay organised through out your research
+4) Supports scheduling jobs / executions to run at specific times, dates and intervals using CRON for automation tasks.
```
Comment (ZihengSun) [inline README.md:13]: I mean you need to add the full explained name of CRON. For people outside computer science, they have no idea what CRON is.

**Review by ZihengSun — CHANGES_REQUESTED** [review]
Another comment

File: README.md [hunk 3]
```diff
@@ -34,6 +42,14 @@ Only two things basically:
 
 [A live demo site](https://geobrain.csiss.gmu.edu/Geoweaver) is available.
 
+
+https://github.com/JYOSHREDDY/Geoweaver/assets/102822880/15bad419-0449-4301-9a56-b49c6106ea82
+
+![Screen Recording 2024-03-16 at 11 19 55 PM](https://github.com/JYOSHREDDY/Geoweaver/assets/102822880/8d6fb615-1430-4fe5-9728-e99586c9fd2a)
```
Comment (gokulprathin8) [inline README.md:48]: The processes in the video are failing. Please use a different worklfow, the SWE workflow has a few dependencies / assets on an external server which require you to download a few gigs of files.

Also, the recording source should be ESIPFed repo, as of now it is coming from your github account. Please verify

**Review by gokulprathin8 — CHANGES_REQUESTED** [review]

**Comment by gokulprathin8** [comment]
lets push this and make additional changes to the issue https://github.com/ESIPFed/Geoweaver/issues/463
and open a new PR to address new changes, this PR has been open for a while.

**Comment by gokulprathin8** [comment]
@JYOSHREDDY 
please open a new PR for https://github.com/ESIPFed/Geoweaver/issues/463

**Pull request merged**